css/main.css

```diff
@@ -0,0 +1,137 @@
+h1 {
+    color: #dedede;
+}
+
+p {
+    color: #eeeeee;
+}
+
+/* Navigation */
+
+.navbar {
+    background: rgb(255, 0, 0, 0.7) !important;
+}
+
+.nav-link,
+.navbar-brand {
+    color: #dedede !important;
+    cursor: pointer !important;
+}
+
+.nav-link {
+    margin-right: 1em !important;
+}
+
+.nav-link:hover {
+    background: rgba(0, 0, 0, 0);
+    color: #af0404 !important;
+}
+
+.navbar-collapse {
+    justify-content: flex-end;
+}
+
+.navbar-toggler {
+    background: rgb(255, 0, 0, 0.7) !important;
+}
+
+.header {
+    background-image: url('../images/bkgtop.png');
+    background-attachment: fixed;
+    background-size: cover;
+    background-position: center;
+    width: 100%;
+    height: 850px;
+}
+
+.overlay {
+    position: absolute;
+    min-height: 100%;
+    min-width: 100%;
+    left: 0;
+    top: 0;
+}
+
+.description {
+    position: absolute;
+    top: 30%;
+    margin: auto;
+    padding: 2em;
+    width: 50%;
+}
+
+.description h1 {
+    color: #af0404;
+    font-size: 3rem;
+}
+
+.description p {
+    color: #dedede;
+    font-size: 1.2rem;
+    line-height: 1.5;
+}
+
+
+/* Community Section*/
+
+.community {
+    background-color: #af0404;
+    padding-bottom: 50px;
+}
+
+.text {
+    margin-top: 20%;
+}
+
+.buttons {
+    margin-top: 20%;
+    display: flex;
+    flex-direction: column;
+
+}
+
+/*Kizomba Section*/
+.kizomba {
+    background-color: #414141;
+    padding-bottom: 30px;
+    padding-top: 5px;
+}
+
+
+.img {
+    margin: auto !important;
+    width: 300px;
+    height: auto;
+    display: block;
+    border-radius: 10px;
+}
+
+
+
+/*Buttons*/
+
+.button {
+    border-radius: 20px;
+    color: #AF0404 !important;
+    font-family: Open Sans;
+    font-size: 26px;
+    font-weight: 100;
+    padding: 20px;
+    background-color: #DEDEDE;
+    opacity: 0.75;
+    box-shadow: 1px 1px 20px 0px #FF4949;
+    text-shadow: 1px 1px 20px #FF4949;
+    border: solid #C10000 1px;
+    text-decoration: none !important;
+    display: inline-block;
+    cursor: pointer;
+    text-align: center;
+    margin: 10px;
+}
+
+.button:hover {
```

Comment:
Don't forget `:focus` state too when building your styles. It's activated when navigating a page using the keyboard.